public/js/whydPlayer.js

```diff
@@ -496,19 +496,31 @@ function WhydPlayer() {
   // handlers for events coming from Playem
 
   var playemEventHandlers = {
-    onError: function(e) {
+    onError: function (e) {
       if (currentTrack.metadata.logData) logTrackPlay();
 
+      // will try to fallback failing tracks to deezer player
+      //Detect user agent for electron specific message
+      var userAgent = navigator.userAgent.toLowerCase();
+      var isElectronApp = false;
+      var failedTrackMessage;
+      if (userAgent.indexOf(' electron/') > -1) {
+        isElectronApp = true;
```

Comment:
If I understand well, the `isElectronApp` is not used anywhere. If so, can we remove that variable?

Reply:
Yeah this variable is not used anywhere else so it can definitely be removed